spec/index.html

```diff
@@ -385,8 +385,8 @@ <h2>Introduction</h2>
       </p>
       <p>
         The SPARQL query language for RDF is designed to meet the use cases and
-        requirements identified by the RDF Data Access Working Group in [[rdf-dawg-uc]],
-        the SPARQL 1.1 Working Group in [[sparql-features]], and the RDF-star Working Group.
+        requirements identified by the RDF Data Access Working Group in [[RDF-DAWG-UC]],
+        the SPARQL 1.1 Working Group in [[SPARQL-FEATURES]], and the RDF-star Working Group.
```

Comment:
Genuine question: do we want these to be normative references?

Reply:
@Tpt Agree. The problem is that Informative references are not generated. I don't know why, but I suggest unifying the capital letter issue, and let's clean the HTML/ReSpec in a different PR.

Reply:
Yes! Task #120